eman2/protocols.conf

```diff
@@ -41,7 +41,12 @@ Random Conical Tilt = [
 Tomography = [
 	{"tag": "section", "text": "Imports", "icon": "bookmark.png", "children": []},
 	{"tag": "section", "text": "Movies/Micrographs", "children": []},
-	{"tag": "section", "text": "Reconstruction", "children": []},
+	{"tag": "section", "text": "Reconstruction", "children": [
```

Comment:
"Reconstruction" section should go after "subtomogram averaging" section, just to follow a typical tomography workflow.

Reply:
Done https://github.com/scipion-em/scipion-em-eman2/pull/36/commits/df1ed7237ecdf2e47a930f08d0b5fe8ee516a487 I also move Particles Picking and Extraction to the top following the same logic